src/udev/udev-builtin-net_id.c

```diff
@@ -163,8 +163,29 @@ static bool is_valid_onboard_index(unsigned long idx) {
         return idx <= (naming_scheme_has(NAMING_16BIT_INDEX) ? ONBOARD_16BIT_INDEX_MAX : ONBOARD_14BIT_INDEX_MAX);
 }
 
+static int parse_vf_repr_num(const LinkInfo *info, unsigned *num) {
+        unsigned pf;
+
+        assert(info);
```

Comment:
Please add assertion for `num`.
Also, we usually name argument for storing result `ret`.